es-ds-docs/pages/molecules/verification-code.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+if ($prism) {
```

Comment:
Need to wrap this whole block in an async function inside onMounted as has been done in other components, for prism to work correctly